packages/api-generator/src/locale/en/v-chip.json

```diff
@@ -17,7 +17,7 @@
     "text": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/next/packages/api-generator/src/locale/en/v-chip.json))",
     "activeClass": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/next/packages/api-generator/src/locale/en/v-chip.json))",
     "appendAvatar": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/next/packages/api-generator/src/locale/en/v-chip.json))",
-    "appendIcon": "MISSING DESCRIPTION ([edit in github](https://github.com/vuetifyjs/vuetify/tree/next/packages/api-generator/src/locale/en/v-chip.json))",
+    "appendIcon": "Adds an icon after the default slot content.",
```

Comment:
Any reason for not updating `prependIcon` and the other quick wins?

Reply:
Hi, I'm sorry about that. This is my first time to send a pull request so I'm not familiar with the process. I love this lib a lot so I try to help. Can I edit this PR to make it completed? Or I need to send another one? Thanks for your reply .